### PR TITLE
Csv import

### PR DESCRIPTION
**Pull request by jschintz**
Manual process to import apps info.  
- Imports csv file of ecosystem entities
- Overloads collect statement to gather csv.

### CODE REVIEW

**Comment by jschintz** [comment]
Manually Import File of Ecosystem Entity Members

File: aggregations/sql_aggregations.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def store(self, parameters: list):
                 except psycopg2.errors.UniqueViolation:
                     self.analytics_connection.rollback()
 
+
```
Comment (frol) [inline aggregations/sql_aggregations.py:65]: Don't introduce unnecessary changes

File: aggregations/db_tables/near_ecosystem_entities.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    @property
+    def sql_insert(self):
+        return '''
+            INSERT INTO near_ecosystem_entities VALUES %s
+            ON CONFLICT DO NOTHING
+        '''  
+
+#overload collect method defined in sql_aggregations
+#    @staticmethod
```
Comment (frol) [inline aggregations/db_tables/near_ecosystem_entities.py:40]: Clean up your code before submitting the code for review

File: aggregations/db_tables/near_ecosystem_entities.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import csv
+from ..sql_aggregations import SqlAggregations
+
+
+class NearEcosystemEntities(SqlAggregations):
+    @property
+    def sql_create_table(self):
+        # All fields are text, of varying length
```
Comment (frol) [inline aggregations/db_tables/near_ecosystem_entities.py:8]: How does this comment help?

**Review by frol — CHANGES_REQUESTED** [review]
* [ ] Before we merge it, can you show how will you use this data? How does it help you to have this raw dump of data in the database?
* [ ] Clean up your PR:
  * [ ] Remove `.DS_Store` files
  * [ ] Address the comments I left at the specific places

**Comment by jschintz** [comment]
Thanks, Vlad

I just adjusted those changes you asked for.
Below is how we intend to use the file.  Would you prefer to talk it
through with myself & Neil?

Thanks,
James

The csv contains information about partners that is currently not on the
chain, and stored in the ecosystem map repo. We are currently working with
the ES team to create a CI that will create this json. The reason for the
temporary method is to unblock the aggregations that are reliant on the app
data.The data will be used to aggregate the accounts data by app as is
needed for 4/5 proposed visualizations for the new stats site. The
ecosystem data contains the contract_ids needed to make this aggregation
and additional information such as the App name, website links etc that
Erik has requested to be included in the visualizations & tooltips. For
reference, please see the designs
<https://www.figma.com/proto/Lfw8WzdlqP63QdQj2C0NwM/stats.near.org-designs?node-id=1%3A441&scaling=min-zoom&page-id=0%3A1>
.Even without the visualizations, any app level aggregations will need to
related to an app name and associated contracts, as some apps have more
than one parent contract listed.The app aggregations will take the form of
a single additional table added to the NEAR analytics database that will
contain daily_accounts_per_app_count in a similar format to
daily_transactions_per_account_count table that has already been created by
the NEAR team.

**Review by frol — APPROVED** [review]
I still don't see how this table will help you in any way given that it mostly lacks the account/contract ids anyway.

@telezhnaya Let's merge it as is, noting to ourselves that this table is a temporary solution.

@jschintz On the next iterations I would prefer to only store cleaned data to the near-analytics database.

**Pull request merged**